### PR TITLE
Add DeltaOm3ga charter icon

### PR DESCRIPTION
**Pull request by DeltaOm3gaCharting**

### CODE REVIEW

**Review by kaduwaengertner — APPROVED** [review]

**Comment by kaduwaengertner** [comment]
I think the only thing that needs change is the type to charter?

File: extra/index.json [hunk 1]
```diff
@@ -1212,6 +1212,14 @@
...
 			"icon": "french",
 			"type": "custom"
+		},
+		{
+			"ids": [ "deltaom3ga" ],
+			"names": {
+				"en-US": "DeltaOm3ga"
+			},
+			"icon": "deltaom3ga",
+			"type": "custom"
```
Comment (EliteAsian123) [inline extra/index.json:1222]: `custom` -> `charter`

**Review by EliteAsian123 — CHANGES_REQUESTED** [review]

File: extra/index.json (hunk 1 above)
Comment (DeltaOm3gaCharting) [reply extra/index.json:1222]: How would I go about doing that?

Comment (EliteAsian123) [reply extra/index.json:1222]: Change `"type": "custom"` to `"type": "charter"`

**Review by kaduwaengertner — APPROVED** [review]

**Review by EliteAsian123 — APPROVED** [review]

**Pull request merged**